boxtree/pyfmmlib_integration.py

```diff
@@ -529,6 +529,8 @@ def get_source_kwargs(self, src_weights, pslice):
     @log_process(logger)
     @return_timing_data
     def form_multipoles(self, level_start_source_box_nrs, source_boxes, src_weights):
+        assert len(src_weights) == 1
+        src_weights = src_weights[0]
```

Comment:
```suggestion
        src_weights, = src_weight_vecs
```
(Same repeated below.)